target/product/emulator.mk

```diff
@@ -20,7 +20,22 @@
 
 # Host modules
 PRODUCT_PACKAGES += \
-
+    emulator \
+    emulator-x86 \
```

Comment:
I suggest to put this line after 'emulator-mips' for alphabetical order. Here and for 'emulator64' below.